schema_utils.py

```diff
@@ -229,47 +262,120 @@ def get(cls, validator_id):
     def has_length_at_least(obj, min_value):
         """Returns True iff the given object (a list) has at least
         `min_value` elements.
+
+        Args:
+            obj: *. Object to compare with min_value.
+            min_value: int. Minimum value of elements.
```

Comment:
min_value is not the minimum value of elements (e.g. if the list is [1, 2, 3] then the minimum value is 1). It is the required minimum number of elements in the list.

(I commented on this before but it does not seem to be fixed. When you reply "Done" please make sure that the comment is actually addressed in the code. This is important in order to reduce review cycles.)

Ditto below for max_value.